### PR TITLE
Print feature working now

### PR DESCRIPTION
**Pull request by chrisvanmook**

### CODE REVIEW

File: modules/atlas/styles/deprecated/_dashboard-print.scss [hunk 1]
```diff
@@ -50,3 +51,30 @@ $dashboard-heading-padding-print-version: 46px;
     border: none;
   }
 }
+
+// Todo: move this logic to js
```
Comment (RRMoelker) [inline modules/atlas/styles/deprecated/_dashboard-print.scss]: Hmm, I would say we have to much presentation logic in JS at the moment. I would rather have everything else moved to css. But that's way to much work. So just remove Todo?

File: modules/atlas/styles/deprecated/_dashboard-print.scss [hunk 2]
```diff
@@ -50,3 +51,30 @@ $dashboard-heading-padding-print-version: 46px;
...
+// sass-lint:disable nesting-depth
+.c-dashboard--page-type {
+  &-search-data {
```
Comment (RRMoelker) [inline modules/atlas/styles/deprecated/_dashboard-print.scss]: Oef... Can't we just make all dashboards columns width 100%? Not just the `search-data` one?

File: modules/header/components/menu/_menu.scss [hunk 3]
```diff
@@ -34,7 +34,7 @@
 
 .c-menu__item--help {
   .c-menu__title {
-    .c-dashboard--page-type-help & {
+    .c-dashboard--page-type--help & {
```
Comment (RRMoelker) [inline modules/header/components/menu/_menu.scss]: These page types are a complete hack... Probably out of scope. But must be said

File: src/app/components/SplitScreen/SplitScreen.jsx [hunk 4]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+/* istanbul ignore next */ // TODO: refactor, test
+const SplitScreen = ({ leftComponent, rightComponent }) => (
```
Comment (RRMoelker) [inline src/app/components/SplitScreen/SplitScreen.jsx]: Good one

File: src/shared/ducks/ui/ui.js [hunk 5]
```diff
@@ -21,12 +20,16 @@ export const initialState = {
 
 export default function UiReducer(state = initialState, action) {
   switch (action.type) {
+    case routing.panorama.type:
+    case routing.dataDetail.type:
+    case routing.detail.type:
     case routing.map.type: {
-      const { embedPreview, embed } = action.meta.query || {};
+      const { embedPreview, embed, print } = action.meta.query || {};
```
Comment (RRMoelker) [inline src/shared/ducks/ui/ui.js]: print from query should be read on EVERY route change I think. So if `action.type` starts with `'reduxRouter/'` then..

File: src/store/redux-first-router.js [hunk 6]
```diff
@@ -32,6 +32,12 @@ export const isMapPage = createSelector(
   (currentPage) => currentPage === PAGES.KAART
 );
 
+export const isPanoPage = createSelector(
+  getCurrentPage,
+  (currentPage) => currentPage === PAGES.PANORAMA
+    || currentPage === PAGES.KAART_PANORAMA
```
Comment (RRMoelker) [inline src/store/redux-first-router.js]: There should be only one Panorama "Page" right now. One of these can be trashed

File: modules/atlas/styles/deprecated/_dashboard-print.scss (hunk 1 above)
Comment (chrisvanmook) [reply modules/atlas/styles/deprecated/_dashboard-print.scss]: More like those page-types selectors should be not in css... or js

File: modules/atlas/styles/deprecated/_dashboard-print.scss (hunk 2 above)
Comment (chrisvanmook) [reply modules/atlas/styles/deprecated/_dashboard-print.scss]: columns has a col-sm-8 etc. classes. so width is now overridden basically, hence the Todo up above ;)

File: modules/header/components/menu/_menu.scss (hunk 3 above)
Comment (chrisvanmook) [reply modules/header/components/menu/_menu.scss]: Totally agreed!

File: src/shared/ducks/ui/ui.js (hunk 5 above)
Comment (chrisvanmook) [reply src/shared/ducks/ui/ui.js]: Good one!

File: src/store/redux-first-router.js (hunk 6 above)
Comment (chrisvanmook) [reply src/store/redux-first-router.js]: Thanks!

File: modules/header/components/menu/_menu.scss (hunk 3 above)
Comment (chrisvanmook) [reply modules/header/components/menu/_menu.scss]: Also, we should eventually get rid of the dashboard component-class...

**Pull request merged**